ports/sciter-js/portfile.cmake

```
@@ -0,0 +1,115 @@
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
+
+if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
+    message(FATAL_ERROR "Sciter only supports Windows Desktop")
+endif()
+
+# header-only library
+set(VCPKG_POLICY_DLLS_WITHOUT_LIBS enabled)
+
+set(SCITER_REVISION b329d51dd28da112d6312ade46e0b68aa5abd0c6)
+set(SCITER_SHA 0fe2c9ed7baf5f9a18f037fff90e91ef9a229270e0b5489b051a4f44230e8123c0883a70f582767c1d7811de439beadbe400e54796bff7ae6cea4f89a65b30dc)
+
+if(VCPKG_TARGET_ARCHITECTURE STREQUAL x64)
+    set(SCITER_ARCH x64)
+elseif(VCPKG_TARGET_ARCHITECTURE STREQUAL x86)
+    set(SCITER_ARCH x32)
+endif()
+
+# check out the `https://github.com/c-smile/sciter-js-sdk/archive/${SCITER_REVISION}.tar.gz`
+# hash checksum can be obtained with `curl -L -o tmp.tgz ${URL} && vcpkg hash tmp.tgz`
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO c-smile/sciter-js-sdk
+    REF ${SCITER_REVISION}
+    SHA512 ${SCITER_SHA}
+)
+
+# install include directory
+file(INSTALL ${SOURCE_PATH}/include/ DESTINATION ${CURRENT_PACKAGES_DIR}/include/sciter-js
+    FILES_MATCHING
+    PATTERN "*.cpp"
+    PATTERN "*.mm"
+    PATTERN "*.h"
+    PATTERN "*.hpp"
+)
+
+set(SCITER_SHARE ${CURRENT_PACKAGES_DIR}/share/sciter-js)
```

```suggestion
set(SCITER_SHARE "${CURRENT_PACKAGES_DIR}/share/sciter-js")
```

It would be nice if all paths were quoted but not blocking over that.